DESCRIPTION

```diff
@@ -49,6 +49,7 @@ RoxygenNote: 7.3.0
 SystemRequirements: GNU make, An ODBC3 driver manager and drivers.
 Collate: 
     'RcppExports.R'
+    'aaa-odbc-data-type.R'
```

Comment:
😞

I wasn't able to find any other places where we needed a workaround to define generics before methods. We could also just delete this file and place its contents in `odbc-connection.R`?